### PR TITLE
Do not gitignore rel diretory

### PR DESCRIPTION
**Pull request by sabondano**

### CODE REVIEW

File: priv/scripts/inject_erlang_cookie.sh [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+cat ./rel/config.exs | sed "s/{ERLANG_COOKIE}/`cat erlang.secret.cookie`/g" > ./rel/config.exs
```
Comment (paulswartz) [inline priv/scripts/inject_erlang_cookie.sh]: Doesn't this already happen if you use `${ERLANG_COOKIE}` as the value and `REPLACE_OS_VARS=true` in the environment?

Comment (elbow-jason) [reply priv/scripts/inject_erlang_cookie.sh]: Oh man. We had the `REPLACE_OS_VARS=true` BELOW the place it was needed.

I'm removing this script and updating the readme

File: README.md [hunk 2]
```diff
@@ -163,20 +163,31 @@ Run every 2 minutes
...
 `docker build -t api-checker:latest .`
 
-To demo this in Docker using dev config, run this command after building:
 
-`docker run -e "API_CHECKER_CONFIGURATION=$(cat ./priv/dev_checks_config.json)" api-checker:latest`
+To demo this in Docker using dev config and using the cookie `a_super_secret_cookie` run this command after building:
+
+Note: the `ERLANG_COOKIE` env var should be secret and secure in production as it allows connection to the running erlang cluster which allows execution of arbitrary code.
+
+`docker run -e "API_CHECKER_CONFIGURATION=$(cat ./priv/dev_checks_config.json)" -e "ERLANG_COOKIE=a_super_secret_cookie" api-checker:latest`
```
Comment (elbow-jason) [inline README.md]: @paulswartz This is how to get the `ERLANG_COOKIE` into the docker container. Good call on the `REPLACE_OS_VARS=true`.

File: priv/scripts/inject_erlang_cookie.sh (hunk 1 above)
Comment (elbow-jason) [reply priv/scripts/inject_erlang_cookie.sh]: Fixed in fdfeaf7665275b6df7339a3ecc88dee62db767a3

File: Dockerfile [hunk 3]
```diff
@@ -15,6 +15,10 @@ ADD . .
 
 WORKDIR /root
 
+# Instructs distillery to interpolate environment variables at runtime.
```
Comment (paulswartz) [inline Dockerfile]: You don't need this here: it's already on line 31. This environment is used at compile time, so this runtime variable doesn't have an effect.

File: .gitignore [hunk 4]
```diff
@@ -21,5 +21,5 @@ erl_crash.dump
 
 .vscode/
 
-# Distillery config
-/rel/
+# Ignore secrets
+*.secret.*
```
Comment (paulswartz) [inline .gitignore]: Are there any secret files now?

File: Dockerfile (hunk 3 above)
Comment (sabondano) [reply Dockerfile]: Removed 👍

File: .gitignore (hunk 4 above)
Comment (sabondano) [reply .gitignore]: We had a cookie in a secret file but if you agree we'll let you generate a cookie as needed and decide how you want to store it.

Removed this line for the time being.

Comment (elbow-jason) [reply .gitignore]: Paul, that's a secret.

Comment (elbow-jason) [reply .gitignore]: Just kidding. There are no secret files anymore. We can remove this.

**Pull request merged**